cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -1369,6 +1370,18 @@ export class CanvasViewImpl implements CanvasView, Listener {
         return this.canvas;
     }
 
+    public destroy(): void {
+        this.canvas.dispatchEvent(
+            new CustomEvent('canvas.destroy', {
+                bubbles: false,
+                cancelable: true,
+            }),
+        );
+        // We can't call namespaced svgjs event
+        // see - https://svgjs.dev/docs/2.7/events/
```

Comment:
What exactly should I see? 

I read:

```
SVG.js supports namespaced events following the syntax event.namespace.

A namespaced event behaves like a normal event with the difference that you can remove it without touching handlers from other namespaces.
```

Reply:
Yes, it says it supports namespaced events, but however we can't call it directly. That's what I've meant here.
In the end of this page there is:
```
However, you can't fire a specific namespaced event. Calling element.fire('myevent.namespace') won't do anything while element.fire('myevent') works and fires all attached handlers of the event
```
Maybe it's better to update this comment this way
```// We can't call a specific namespaced event ```